<launch>
    <arg name="visualization" default="true"/>
    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>
    <node pkg="adaptive_viewutility" type="test_grid_node" name="test_grid" output="screen">
        <param name="file_path" value="/home/jaeyoung/dev/mesh/groundtruth_roi_meshlab.obj"/>
        <param name="num_experiments" value="10"/>
        <param name="result_directory" value="$(find adaptive_viewutility)/../output"/>
        <param name="max_experiment_duration" value="1000"/>
    </node>

    <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
        <rosparam command="load" file="$(find adaptive_viewutility)/launch/visualize.yaml" />
    </node>

    <group if="$(arg visualization)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find adaptive_viewutility)/launch/config_planner.rviz" />
    </group>
</launch>
